README.md

```diff
@@ -25,16 +33,16 @@ in turn to the callback:
 // => [1, 4, 9]
 ```
 
-While both `.filter()` and `.map()` return a new array, `.filter()` returns a
+While both `filter()` and `map()` return a new array, `filter()` returns a
 subset of the original array (unless all elements meet the provided condition)
-in which the elements are unchanged. `.map()`, on the other hand, returns a new
+in which the elements are unchanged. `map()`, on the other hand, returns a new
 array that's the same length as the original array in which the elements have
 been modified.
 
-Let's quickly run through how we could create our own version of the `.map()`
+Let's quickly run through how we could create our own version of the `map()`
 method.
 
-## Implementing `.map()` From Scratch
+## Implementing `map()` From Scratch
 
 ### Abstracting the iteration
```

Comment:
Noticed this in other lessons I've reviewed as well, but it didn't seem like it would be an issue until now - I believe when these (`###` headers) get converted to canvas lessons, they all unfortunately become headers of the same size, rather than creating a discernible sub-header. 

So it might turn out to look like: 

## Implementing `map()` from scratch

## Abstracting the iteration 

Which could look as if there's content missing under the first header. Might be worth double-checking though